src/apps/transfer_ownership/api.py

```diff
@@ -31,6 +33,10 @@ async def transfer_accept(
     """Respond to a transfer of ownership of an applet."""
     async with atomic(session):
         await TransferService(session, user).accept_transfer(applet_id, key)
+        try:
+            await ScheduleService(session).delete_all_schedules(applet_id)
+        except NotFoundError:
+            pass
```

Comment:
Dont we need to move logic of deleting schedule inside TransferService(session, user).accept_transfer() ?

Reply:
I don't see any reason to duplicate the code from one service to another. Schedule service incapsulates the logic of deletion of events, so why not to use it.